week-1-ts-ecosystem/w01e01-typescript-basics/src/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from './types';
+
+export const hasAddress = (user: User): boolean => {
+  return Boolean(user.address);
+};
+
+export const hasGivenAge = (requiredAge: number): ((user: User) => boolean) => {
+  return (user: User): boolean => user.age >= requiredAge;
+};
+
+export const isAdult = hasGivenAge(18);
```

Comment:
Ten plik nazwałbym inaczej 

`utils.ts` jest zbyt ogólne.
Jeśli bardzo chcesz iść w te strone to: `user-utils.ts`

a lepiej moim zdaniem np.:
`user.validators.ts`